.github/workflows/pr.yml

```diff
@@ -34,3 +34,23 @@ jobs:
           go-version: stable
           cache: true
       - run: "go test ./... -v"
+  grafana-integration-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [main, v11.2.x, v11.1.x, v11.0.x, v10.4.x, v10.3.x]
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Checkout grafana
+        uses: actions/checkout@v4
+        with:
+          repository: grafana/grafana
+          ref: ${{ matrix.version }}
+          path: grafana-oss
```

Comment:
nitpick but can we just call it `grafana`? the repo is just `grafana`. :joy:

Reply:
i was mostly thinking that there's less chance of a name collision with code committed to the repo. i'll rename